site/src/Librecores/ProjectRepoBundle/Controller/ProjectController.php

```diff
@@ -44,12 +46,12 @@ public function newAction(Request $request)
 
         // construct choices for the owner
         $username = $this->getUser()->getUsername();
-        $parentChoices = array($username => 'u_' . $username);
+        $parentChoices = array($username => 'u_'.$username);
         foreach ($this->getUser()->getOrganizationMemberships() as $organizationMembership) {
             if ($organizationMembership->getPermission() === OrganizationMember::PERMISSION_MEMBER or
```

Comment:
There appears to be a difference between `and`,`or` and `&&',`||`. Let's standardize on what to use.

http://php.net/manual/en/language.operators.logical.php

Reply:
Good catch. `||` is the more common one in my world as it has the same precedence as in C (especially regarding the ternary operator). I didn't find a phpcs rule for that and I don't think it's worth writing one, so I've just updated the docs and the code.